### PR TITLE
Add luke to members

### PR DESCRIPTION
**Pull request by Cravey0053**
Adds Luke Cravey to People tab. Includes profile text and profile image

### CODE REVIEW

File: _members/cravey-luke.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Luke Cravey
+image: images/cravey-luke.jpg
+role: undergrad
```
Comment (mphoward) [inline _members/cravey-luke.md:4]: We changed the name of this role on the `main` branch, so this should become:
```suggestion
role: undergrad-student
```

**Review by mphoward — CHANGES_REQUESTED** [review]
Thanks Luke! This looks great. Can you please merge `main` into this branch, and then apply the change below? I will merge after that.

**Comment by Cravey0053** [comment]
> Thanks Luke! This looks great. Can you please merge `main` into this branch, and then apply the change below? I will merge after that.

Done! Should be good to go.

**Review by mphoward — APPROVED** [review]
All good, thank you!

**Pull request merged**